core/example/CMakeLists.txt

```diff
@@ -3,5 +3,12 @@ if(NOT Cabana_ENABLE_EXAMPLES)
   return()
 endif()
 
+find_package(PkgConfig)
+pkg_check_modules(SCAFACOS scafacos IMPORTED_TARGET)
```

Comment:
Why did you chose this over `find_package(scafacos)`?

Reply:
Also did you consider having an option `Cabana_ENABLE_SCAFACOS` that defaults to false and adding the `REQUIRED` argument when you look for it.

Reply:
I am usually more a fan of "enable everything you can automatically". CMake has the `CMAKE_DISABLE_FIND_PACKAGE_<FOO>=ON` options to disable stuff.

Reply:
> Why did you chose this over find_package(scafacos)?

You didn't address that comment

Reply:
I was just too lazy to write a FindScafacos.cmake module.

Reply:
Not sure, I don't see Scafacos providing any CMake files!

Reply:
I got it, v1.0.1 doesn't have these CMake files, but we can change the once scafacos 1.1 got released.

Reply:
@mh0fm4, when is the next release of scafacos planned?